torchrl/envs/transforms/transforms.py

```diff
@@ -2761,3 +2759,79 @@ def transform_observation_spec(self, observation_spec: TensorSpec) -> TensorSpec
                 if key in self.selected_keys
             }
         )
+
+
+class TimeMaxPool(Transform):
```

Comment:
I think that in the case where out_keys is not in_keys we'll have a problem with the specs.
Can you address this? We just need to copy the spec of all the in_keys on the out_keys